# commit message copied from openembedded:
#    commit 246c80637b135f3a113d319b163422f98174ee6c
#    Author: Khem Raj <raj.khem@gmail.com>
#    Date:   Wed Jun 9 13:37:03 2010 -0700
#
#    shadow-4.1.4.2: Add patches to support dots in login id.
#    
#    Signed-off-by: Khem Raj <raj.khem@gmail.com>
#
# comment added by Kevin Tian <kevin.tian@intel.com>, 2010-08-11

http://bugs.gentoo.org/show_bug.cgi?id=301957
https://alioth.debian.org/scm/browser.php?group_id=30580

Upstream-Status: Pending

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

Index: shadow-4.1.4.2/src/su.c
===================================================================
--- shadow-4.1.4.2.orig/src/su.c	2009-07-23 13:38:56.000000000 -0700
+++ shadow-4.1.4.2/src/su.c	2010-06-03 17:46:47.718944010 -0700
@@ -378,7 +378,7 @@ int main (int argc, char **argv)
 #endif
 #endif				/* !USE_PAM */
 
-	sanitize_env ();
+	/* sanitize_env (); */
 
 	(void) setlocale (LC_ALL, "");
 	(void) bindtextdomain (PACKAGE, LOCALEDIR);
